syntax/JSONC.sublime-syntax

```diff
@@ -0,0 +1,143 @@
+%YAML 1.2
```

Comment:
What's the point of this sublime-syntax file?

Reply:
He doesn't want this plugin to depend on `PackageDev` so he creates it for sublime-related files.

Reply:
Yes, that is the point. 

> LSP-json will have its own JSONC syntax file that will come with this package, and will be assigned to all sublime, settings, theme, completions, keymaps, files... So no need to have 10 different scopes in the config for jsonc. There will be just one and that is source.jsonc.

Reply:
I don't like it :( The PackageDev syntaxes have great highlighting. What's more is that there is already a syntax for JSON with comments at Packages/JavaScript/JSON.sublime-syntax.
Why does this plugin need to depend on PackageDev in the first place?
Also, why not write a language server for json-with-comments?

Reply:
Why not put `jsonc` instead of `json` for the languageId? Wouldn't that make the json-language-server shut up about comments? https://github.com/sublimelsp/LSP-json/blob/master/plugin.py#L85

https://github.com/vscode-langservers/vscode-json-languageserver#server-capabilities

> jsonc documents additionally accept single line (//) and multi-line comments (/* ... */).

Reply:
I see now that this pull request modifies the languageId from `json` to `jsonc`. So that should already remove the errors about comments. So then why not use Packages/JavaScript/JSON.sublime-syntax? Remember that maintaining a custom JSON syntax is some serious maintainer burden...

Reply:
> Remember that maintaining a custom JSON syntax is some serious maintainer burden...

I don't like this solution either (maybe because I have PackageDev installed :D ), but...

> So then why not use Packages/JavaScript/JSON.sublime-syntax?

ST's JSON doesn't not differentiate `json` and `jsonc`. If we do so, LSP will not treat a comma in a JSON file as an error.